styles.css

```diff
@@ -0,0 +1,3 @@
+H1{
+    color: red;
```

Comment:
Rood is wel een donkere kleur als H1. ook had dit h1 moeten zijn (zo heet de tag ook op regel 9 van je index.html